core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -683,10 +668,17 @@ public void setBlockBuildWhenUpstreamBuilding(boolean b) throws IOException {
         save();
     }
 
+    @Override
     public boolean isDisabled() {
         return disabled;
     }
 
+    @Restricted(DoNotUse.class)
```

Comment:
huh? It will allow the parents method, but not the implementation if we operate with  AbstractProject directly